lib/commands/gesture.js

```diff
@@ -24,21 +24,96 @@ commands.click = async function (el) {
   }
 };
 
+function isSameGestures (gestures, candidates) {
+  if (gestures.length !== candidates.length) {
+    return false;
+  }
+  for (let i in gestures) {
```

Comment:
we don't use `for ... in`, we use `for ... of`; or in the case of wanting the index, just a regular for loop:

```js
for (let i = 0; i < gestures.length; i++) {
```